sources/tests/Unit/Command/InspectSchema.php

```diff
@@ -37,6 +37,9 @@ public function testExecute()
                 'command'          => $command->getName(),
                 'config-name'      => 'pomm_test',
                 'schema'           => 'pomm_test',
+            ],
+            [
+                'decorated' => false
```

Comment:
@chanmix51 
This option disables output coloring ([link](https://github.com/symfony/Console/blob/master/Tester/CommandTester.php#L48)). I added it because the output of the tested commands contains raw escape sequences if the tests launched inside a console emulator supporting ANSI colors, as mentioned [here](http://symfony.com/doc/current/components/console/introduction.html#coloring-the-output).